lib/certifications/versions/1_0_0/certification.go

```diff
@@ -0,0 +1,39 @@
+package certification
+
+import (
+	"sort"
+	"vbom.ml/util/sortorder"
+	v1standards "github.com/opencontrol/compliance-masonry/lib/standards/versions/1_0_0"
+	"github.com/opencontrol/compliance-masonry/lib/common"
+)
+
+// Certification struct is a collection of specific standards and controls
+// Schema info: https://github.com/opencontrol/schemas#certifications
+type Certification struct {
+	Key       string              `yaml:"name" json:"name"`
```

Comment:
I saw this when I was moving around the code. Ideally, we should to prevent confusion. Will make a separate PR for this.